src/components/include/utils/logger.h

```diff
@@ -43,6 +43,7 @@
 #endif
 
 #include "utils/pimpl.h"
+#include "../utils/include/utils/appenders_loader.h"
```

Comment:
@OHerasym I think we have to move `appenders_loader.h` to common `include` in case we need it here because common `includes` should not be dependent on components.